systers_portal/systers_portal/settings/dev.py

```diff
@@ -7,8 +7,8 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'systersdb',
-        'USER': '',
-        'PASSWORD': '',
+        'USER': 'postgres',
+        'PASSWORD': 'akanksha',
```

Comment:
The password for your local postgres server is "akanksha". This kind of stuff shouldn't appear in commits. Again please don't do `git add.`.